addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlUtils.java

```diff
@@ -0,0 +1,194 @@
+                params.add(StringUtils.strip(line));
+            }
+            logger.info("Succesfully parsed keys!");
+        } catch (Exception e) {
+            logger.error(e);
+        }
+        return params;
+    }
+
+    /**
+     * Returns "crashproof" HTTP Responses from a given URL by using GET method.
+     *
+     * @param url the url to be tested.
+     * @param headers the associated headers.
+     * @return a HttpMessage of the HTTP Response from the given URL.
+     */
+    public static HttpMessage stableRequest(String url, HttpRequestHeader headers) {
+        try {
+            URI parsed = new URI(url, true);
+            String scheme = parsed.getScheme();
+            String host = parsed.getHost();
+            String path = parsed.getPath();
+            String port = parsed.getPort() == -1 ? "" : Integer.toString(parsed.getPort());
+            String[] schemes =
+                    (scheme.equalsIgnoreCase("https"))
+                            ? new String[] {"https", "http"}
+                            : new String[] {"http", "https"};
+            int errorCount = 0;
+            for (String s : schemes) {
+                try {
+                    HttpMessage msg = new HttpMessage();
+                    headers.setURI(new URI(s + "://" + host + ":" + port + path, true));
+                    headers.setMethod("GET");
+                    msg.setRequestHeader(headers);
+                    sender.sendAndReceive(msg);
+                    return msg;
+                } catch (Exception e) {
+                    // Check if it is a connection error
+                    if (e.getMessage().contains("ConnectionError")
+                            || e.getMessage().contains("IOException")) {
+                        continue;
+                    }
+                    logger.info(e);
+                    errorCount = errorCount + 1;
+                }
+            }
+            logger.info("Total number of Errors: " + errorCount);
+        } catch (URIException e) {
+            logger.error("Error parsing URL: {} Exception raised {}", url, e);
+        }
+        return null;
+    }
+
+    /**
+     * Returns a Map of all parameters to be used from a given wordlist along with a fuzzy number
+     * key.
+     *
+     * @param list the List of all parameters to be used from a given wordlist.
+     * @return a Map<String, String> of all parameters to be used from a given wordlist along with a
+     *     fuzzy number key.
+     */
+    public static Map<String, String> populate(List<String> list) {
+        Map<String, String> map = new HashMap<>();
+        String key;
+        String s;
+        for (int i = 0; i < list.size(); i++) {
+            s = list.get(i);
+            key = StringUtils.repeat("1", (6 - Integer.toString(i).length())) + Integer.toString(i);
+            map.put(s, key);
+        }
+        logger.info("Populated map with " + map.size() + " keys!");
+        return map;
+    }
```

Comment:
The values here could be more meaningful.
Isn't the map being filled with "word, number"? In which case for the javadoc "key" should actually be "value" or something else.....what's the number used for? In the code 's' could be "word" or "paramName", 'i' could be "idx" or "index".

Also I understand info logging is helpful while building but you can probably change that one to debug at this point.

You can always turn it back on with a JS standalone script (inside ZAP) with something like:
```js
var Configurator = Java.type("org.apache.logging.log4j.core.config.Configurator");
var Level = Java.type("org.apache.logging.log4j.Level");

// https://logging.apache.org/log4j/2.x/javadoc.html
Configurator.setLevel({
  "org.parosproxy.paros.core.scanner.Analyser" : Level.DEBUG,
});
```